apps/vr-tests-react-components/src/stories/Tooltip.stories.tsx

```diff
@@ -79,6 +79,20 @@ storiesOf('Tooltip Converged', module)
         <button>Target</button>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28264&commitId=350ba9e103a6adc92752423b6cf9762a4b3ab339&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1946 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|0| Added|
|Tooltip Converged.overflow-wrap.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>